client/app.jsx

```diff
@@ -36,7 +53,32 @@ export default class App extends React.Component {
     this.setState({ watchlist: entries });
   }
 
+  OpenMenu() {
+    if (this.state.menuOpen === true) {
+      this.setState({ menuOpen: false });
+    } else {
+      this.setState({ menuOpen: true });
+    }
+  }
+
+  OpenWatchlist() {
+    this.setState({ watchlistOpen: true });
+  }
+
+  GoHome() {
+    this.setState({ watchlistOpen: false });
+  }
```

Comment:
I noticed that you are capitalizing the first letter of all of your method names. That's not usually how we do it. Typically in JavaScript, methods are `lowerCamelCase`.